autorag/utils/util.py

```diff
@@ -310,3 +310,10 @@ def sort_by_scores(row, reverse=True):
     results = sorted(zip(row['contents'], row['ids'], row['scores']), key=lambda x: x[2], reverse=reverse)
     reranked_contents, reranked_ids, reranked_scores = zip(*results)
     return list(reranked_contents), list(reranked_ids), list(reranked_scores)
+
+
+def select_top_k(row, top_k: int):
```

Comment:
For making more extendable function, you can get a column names that you want to select top-k as parameter, and return all result, regardless of column names' size.